src/pages/starter-kit/reference-app.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Reference applications
+description: Review reference applications created using the Adobe Commerce integration starter kit.
+keywords:
+ - Extensibility
+ - App Builder
+ - API Mesh
+ - Events
+ - REST
+ - Tools
+---
+
+# Reference Applications
+
+This page provides developers with a basic understanding of the types of applications they can create with Adobe Commerce integration starter kit.
```

Comment:
Are the reference applications templates for creating new applications?  Consider clarifying the description to mention that.